echopype/calibrate/calibrate_ek.py

```diff
@@ -407,6 +407,10 @@ def _get_power_from_complex(
             EchoData["Sonar/Beam_group1"] with selected channel subset
         chirp : dict
             a dictionary containing transmit chirp for BB channels
+        z_et : float
```

Comment:
How about adding `float` type hints to `z_et` and `z_er`, while you're at it? Especially given that the other function arguments already have type hints.

Reply:
Done!